crates/re_space_view_time_series/src/aggregation.rs

```diff
@@ -147,9 +148,11 @@ impl MinMaxAggregator {
 
             match self {
                 MinMaxAggregator::MinMax => {
-                    aggregated.push(acc_min);
-                    // Don't push the same point twice.
-                    if j > 1 {
+                    if acc_min == acc_max {
+                        // Don't push the same point twice.
+                        aggregated.push(acc_min);
+                    } else {
+                        aggregated.push(acc_min);
```

Comment:
Why the change here? Does this actually modify behavior in any way? Comparing `PlotPoint`s in general is shady and I'd rather avoid if if there's no algorithmic reason for it.